# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: modules/libpref/init/StaticPrefList.yaml
# Commit: 380eb92fd801
# Full Hash: 380eb92fd801cf72d4caa01a2761bc7e4c1f9d5f
# Author: James Teh <jteh@mozilla.com>
# Date: 2020-03-27 04:48:07
# Regressor Bug: 1622699
# File Overlap Count: 1
# Description:
#   Bug 1622699 part 3: Remove accessibility.xpcom.traverse_outerdoc pref. r=yzen
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D68304
# ==============================================================================

diff -r b6538a37a0cc -r 380eb92fd801 modules/libpref/init/StaticPrefList.yaml
--- a/modules/libpref/init/StaticPrefList.yaml	Thu Mar 26 18:01:19 2020 +0000
+++ b/modules/libpref/init/StaticPrefList.yaml	Thu Mar 26 18:01:37 2020 +0000
@@ -184,11 +184,6 @@
   value: false
   mirror: always
 
-- name: accessibility.xpcom.traverse_outerdoc
-  type: bool
-  value: false
-  mirror: always
-
 #ifdef ANDROID
   #---------------------------------------------------------------------------
   # Prefs starting with "android."
